corehq/apps/es/queries.py

```diff
@@ -0,0 +1,38 @@
+import re
+
+
+def match_all():
+    """
+    No-op query used because a default must be specified
+    """
+    return {"match_all": {}}
+
+
+def _smart_query_string(search_string):
+    special_chars = ['&&', '||', '!', '(', ')', '{', '}', '[', ']', '^', '"',
```

Comment:
Realize this was just moved, but it seems like a regex would be more concise and be a little clearer. Just personal preference though.